src/ucp/rndv/proto_rndv.c

```diff
@@ -440,7 +440,7 @@ void ucp_proto_rndv_rts_query(const ucp_proto_query_params_t *params,
 
     attr->is_estimation  = 1;
     attr->max_msg_length = SIZE_MAX;
-    attr->lane_map       = UCS_BIT(rpriv->lane);
+    attr->lane_map       = remote_attr.lane_map | UCS_BIT(rpriv->lane);
```

Comment:
To enable all used worker interfaces.
E.g. to enable `sysv/memory`, `knem/memory`, `rc_mlx5/mlx5_0:1` in case of this selection:
```
+---------------------------+---------------------------------------------------------------------------------------------------+
| 0x9976d0 intra-node cfg#1 | tagged message by ucp_tag_send*(fast-completion) from host memory                                 |
+---------------------------+--------------------------------------------------+------------------------------------------------+
|                     0..92 | eager short                                      | sysv/memory                                    |
|                  93..8248 | eager copy-in copy-out                           | sysv/memory                                    |
|              8249..262143 | multi-frag eager copy-in copy-out                | sysv/memory                                    |
|                 256K..inf | (?) rendezvous zero-copy flushed write to remote | 68% on knem/memory and 32% on rc_mlx5/mlx5_0:1 |
+---------------------------+--------------------------------------------------+------------------------------------------------+
```

Otherwise, only `sysv/memory` would be activated.

Reply:
aren't we selecting the lane for sending rts message only?

Reply:
Sorry, I don't get the question. What do you mean by "selecting lane"?
We select rendezvous protocol for big messages in the example. We use lane corresponding to `sysv/memory` to send RTS message. We guess remote protocol during rendezvous protocol initialization, and the remote protocol uses lanes corresponding to `knem/memory`, `rc_mlx5/mlx5_0:1` interfaces.
Do you mean that there is no need to activate `knem/memory`, `rc_mlx5/mlx5_0:1` interfaces?

Reply:
According to offline discussion: we have to activate only the interface corresponding to lane used to send RTS message.

Reply:
Fixed by https://github.com/openucx/ucx/pull/8986/commits/e7f37077ea9c0c5b672d3a55b9dea4fc8b54c419